megatron/neox_arguments/neox_args.py

```diff
@@ -709,6 +709,11 @@ class NeoXArgsTraining(NeoXArgsTemplate):
     Output directory to save checkpoints to.
     """
 
+    config_files: dict = None
```

Comment:
I'm wondering if we should initialize non-configurable variables like this in __post_init__ - I don't think they really warrant a docstring, so it might be a better place for them.

Reply:
My thinking was to facilitate development for all not having seen this PR by making all properties of neox args transparent. If some contributor wants to use this info it is nice to read it in the docstring and not be required to read through code to actually see that it exists and what it is.

Reply:
Ah also: the way it is implemented the file contents are part of neox args and distributed by the deepspeed launcher. This means it needs to be a named argument in the current implementation. So the property needs to be defined and then probably also be documented.